xml/crm.dtd

```diff
@@ -65,7 +65,7 @@ Additionally, num_peers and have_quorum are used during the election process to
  * ccm_transition stores the membership instance from the ConsensusClusterMembership layer.
  * cib_feature_revision is the feature set that this configuration requires
 -->
-<!ELEMENT configuration (crm_config, nodes, resources, constraints)>
+<!ELEMENT configuration (crm_config, nodes, resources, constraints, alarms)>
```

Comment:
I looked at this more ... crm.dtd and crm-transitional.dtd are essentially pacemaker-0.x.rng. They predate the rng versioning, but they are used similarly by pacemaker code, i.e. for validation and upgrading of CIB syntax (search for "schema" in lib/common/xml.c). So we do need to keep them around for now, and certainly not modify them.

0.6.7 (the last released version to use DTDs) was released in 2008, so perhaps we can think about retiring them, but I don't think it's worth the bother at this point. 2018 would be a nice round 10 years. I'm thinking that might be a good opportunity to bump to 1.2.0, with the bump justified as losing compatibility with pacemaker < 1.0 (and there's probably a good bit of legacy code that could be removed, too).

Unrelated, I noticed that xml.c uses floats to represent XML versions. We need to change that before hitting x.10 ...